projekt_helm/fiszki/values.yaml

```diff
@@ -2,13 +2,16 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 1
+replicaCount: 3 
+
+namespace:
+  name: lab
 
 image:
-  repository: nginx
-  pullPolicy: IfNotPresent
+  repository: backend-image:latest
+  pullPolicy: IfNotPresent # zostawiłem tak jak było ze wzgledu na najbezpieczniejszą opcję ze wzgledu na błędy występujące z ImagePullBackOff występujące dla wszyskich innych oprócz Never (fajnie by było gdyby obraz był na docker hub)
```

Comment:
Jako spojler alert w przyszłym roku będzie, pomoże to wtedy pozbyć się problemów z tego roku :P